### DISCUSSION TITLE
SO Standalone on Hyper V Host - No Alerts

### DISCUSSION THREAD

**Discussion by techn00** [Security-Onion-Solutions/securityonion#8094] [Unsupported Versions]
I have installed Security Onion-standalone latest on Hyper-V with dedicated virtual switch for monitoring. I am not seeing any alerts in the alerts interface other than the system itself. No error found at all. All services are working. Tried to update to run so-rule update but didn't help.

Any recommendation and suggestions please?

**Comment by dougburks** [comment]
https://docs.securityonion.net/en/2.3/suricata.html#troubleshooting-alerts
